blocks/core/ff_module/ff_module-form-preview/ff_module-form-preview.md

```diff
@@ -25,3 +25,6 @@ data:
 requires: 
     - ff_module-inline-edit            
 ---
+## React Component
+
+<div data-ff_module-form-preview="" ></div>
```

Comment:
Can you add a bit of detail to the md file regarding proptypes (see `ff_module-dropdown-button-component` for examples), and maybe some code-formatted flat html to have an example (I've started adding these in my modal-dialog branch if you want to see what I mean) as this one is not so straightforward layout-wise.